src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -51,38 +43,98 @@ export function ErrorBoundaryInner({
 
 interface ErrorBoundaryFallbackProps {
     error: Error;
-    useRetry?: boolean;
-    resetErrorBoundary?: () => void;
-    onReportProblem?: (error?: Error) => void;
 }
-export function ErrorBoundaryFallback({
-    error,
-    resetErrorBoundary,
-    useRetry,
-    onReportProblem,
-}: ErrorBoundaryFallbackProps) {
+export function ErrorBoundaryFallback({error}: ErrorBoundaryFallbackProps) {
+    const [diagnosticsData, setDiagnosticsData] = React.useState<DiagnosticsData | undefined>();
+
+    React.useEffect(() => {
+        collectDiagnosticsData(error).then((data) => {
+            setDiagnosticsData(data);
+        });
+    }, [error]);
+
+    return (
+        <Flex direction="column" gap={4} className={b(null)}>
+            <Flex direction="row" alignItems="center" gap={10}>
+                <InternalError width={230} height={230} />
```

Comment:
maybe it would be better to use Illustration from our components for consistency

and to create task to move to gravity-ui/illustrations library in the future

just an opinion

Reply:
I don't think that our illustration really suits here. As for the migration - we can replace only access denied svg, other are not present in gravity package